protractor/e2e/integration/secong-page.spec.ts

```diff
@@ -59,4 +59,24 @@ describe('tests for second page', ()=> {
         expect(await secondPage.infoSubmitButton.isDisplayed()).toBeTruthy();
         await secondPage.infoSubmitButton.click();
     });
-});
+
+
+    fit('Homework 2: check amount of articles in the Blog', async () => {
```

Comment:
don't forget to remove "fit" from your tests before pushing it to the repository. Just imagine, that you have 1000 tests in your app and now in your CI will be only 1 test ran:-)